mmv1/templates/terraform/examples/document_ai_warehouse_location.tf.erb

```diff
@@ -0,0 +1,3 @@
+resource "google_document_ai_warehouse_location" "<%= ctx[:primary_resource_id] %>" {
+    location = "us"
+}
```

Comment:
This is an example to use project number. https://github.com/GoogleCloudPlatform/magic-modules/blob/main/mmv1/templates/terraform/examples/pubsub_lite_topic_basic.tf.erb#L3